README.md

```diff
@@ -240,21 +240,126 @@ Similar to providers, plugins are independent projects that conform to the Dayto
+
+   2. **Create a new issue**
+      * Navigate to Issues: Go to the Issues tab [here](https://github.com/daytonaio/daytona/issues).
+      * Click on "New Issue": Choose the appropriate template (Bug Report, Feature Request, etc.) if available.
+      * Fill Out the Issue Template: Provide a clear and concise description of the issue, including steps to reproduce (for bugs) or detailed feature descriptions.
+      * Submit the Issue: Click "Submit new issue" to create the issue.
+   3. Engage with the Community
+      * **Respond to Feedback:** Be prepared to provide additional information or clarification if maintainers or other contributors have questions.
+      * **Collaborate on Solutions:** If you have ideas for resolving the issue, share them in the comments.
+
+### 2. Contributing Code
+
+If you're interested in contributing code to Daytona, follow these steps:
+
+1. **Fork the Daytona repository**
+
+    [Fork](https://github.com/daytonaio/daytona/fork) the GitHub repository to create your own copy of the repository.
+  
+2. **Create a Workspace with Daytona**
+
+    Use the Daytona CLI to create a workspace for your forked repository. Replace YOUR-FORK-URL with the URL of your forked repository.
+    ```bash
+    daytona create YOUR-FORK-URL
+    ```
+3.  **Create a new branch**
+    
+    Once in the development conatiner, create a new branch for your changes:
+    ```bash
+    git checkout -b my-new-feature
+    ```
+
+4. **Run the Project Locally**
+    ```bash
+    dtn serve
+    ```
+
+5. **Make changes to the project**
+
+    Prepare your changes and ensure your commits are descriptive. The document contains an optional commit template, if desired.
 
+6. **Test your changes**
 
+    Ensure to test your changes by running the project locally. You can also run the tests by running `go test ./...` in the root of the project.
+
+7. **Generate docs**
+
+    Ensure to generate new docs after making command related changes, by running ./hack/generate-cli-docs.sh in the daytona root directory.
+
+    ```bash
+    ./hack/generate-cli-docs.sh
+    ```
+
+8. **Generate new API client**
+
+    Ensure to generate a new API client after making changes related to the API spec, by running ./hack/swagger.sh in the daytona root directory.
+
+    ```bash
+    ./hack/swagger.sh
+    ```
+
+9. **Check for lint errors**
+
+    Ensure that you have no lint errors. We use golangci-lint as our linter which you can install by following instructions found [here](https://golangci-lint.run/usage/install/) (or simply open Daytona in a Dev Container). You can check for linting errors by running golangci-lint run in the root of the project.
+
+    ```bash
+    golangci-lint run
+    ```
+
+10. **Sign off on your commits**
+
+    Ensure that you sign off on all your commits to comply with the DCO v1.1. We have more details in [Prepare your changes](#prepare-your-changes).
```

Comment:
The link leads nowhere.

Also, inform that the sign off can be added with `-s` when doing `git commit` which is simpler